pkg/options/options.go

```diff
@@ -164,6 +166,8 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 		"Excluded CIDRs are used to exclude IPVS rules from deletion.")
 	fs.BoolVar(&s.GlobalHairpinMode, "hairpin-mode", false,
 		"Add iptables rules for every Service Endpoint to support hairpin traffic.")
+	fs.Uint16Var(&s.GoBGPAdminPort, "gobgp-admin-port", defaultGoBGPAdminPort,
+		"Port to connect to GoBGP for administrative purposes.")
```

Comment:
Worth mentioning that 0 = disabling it?

Reply:
That didn't really work before, but I made it work now. There have been several requests in the past to not have the GoBGP gRPC port exposed, so hopefully this makes that more accessible for those use-cases.

Thanks for bringing this up!